gbpservice/nfp/configurator/agents/loadbalancer_v2.py

```diff
@@ -0,0 +1,978 @@
+                lb = obj_dict['listener']['loadbalancer']['id']
+            else:
+                # Pool Member or Health Monitor
+                lb = obj_dict['pool']['loadbalancer']['id']
+            # For Liberty
+            # if obj_type == 'loadbalancer':
+            #     lb = obj_dict['id']
+            # elif obj_type == 'listener':
+            #     lb = obj_dict['loadbalancer']['id']
+            # elif obj_type == 'pool':
+            #     lb = obj_dict['listener']['loadbalancer']['id']
+            # elif obj_type == 'sni':
+            #     lb = obj_dict['listener']['loadbalancer']['id']
+            # else:
+            #     # Pool Member or Health Monitor
+            #     lb = obj_dict['pool']['listener']['loadbalancer']['id']
+        except Exception:
+            raise exceptions.IncompleteData(
+                'Root loadbalancer id was not found')
+        else:
+            return lb
+
+    def handle_event(self, ev):
+        """Processes the generated events in worker context.
+
+        Processes the following events.
+        - create loadbalancer
+        - update loadbalancer
+        - delete loadbalancer
+        - create listener
+        - update listener
+        - delete listener
+        - create pool
+        - update pool
+        - delete pool
+        - create member
+        - update member
+        - delete member
+        - create pool health monitor
+        - update pool health monitor
+        - delete pool health monitor
+        - agent updated
+        Enqueues responses into notification queue.
+
+        Returns: None
+
+        """
+        msg = ("Handling event=%s" % (ev.id))
+        LOG.info(msg)
+        try:
+            msg = ("Worker process with ID: %s starting "
+                   "to handle task: %s of topic: %s. "
+                   % (os.getpid(), ev.id, lb_constants.LBAAS_AGENT_RPC_TOPIC))
+            LOG.debug(msg)
+
+            method = getattr(self, "_%s" % (ev.id.lower()))
+            method(ev)
+        except Exception as err:
+            msg = ("Failed to perform the operation: %s. %s"
+                   % (ev.id, str(err).capitalize()))
+            LOG.error(msg)
+        finally:
+            if ev.id == lb_constants.EVENT_COLLECT_STATS:
+                """Do not say event done for collect stats as it is
+                   to be executed forever
+                """
+                pass
+            else:
+                msg = ("Calling event done for event=%s" % (ev.id))
+                LOG.info(msg)
+                self.sc.event_done(ev)
+
+    def _handle_event_loadbalancer(self, ev, operation):
+        data = ev.data
+        context = data['context']
+        loadbalancer = data['loadbalancer']
+        root_lb_id = self._root_loadbalancer_id('loadbalancer', loadbalancer)
+        agent_info = ev.data['context'].pop('agent_info')
+        service_vendor = agent_info['service_vendor']
+
+        try:
+            if operation == 'create':
```

Comment:
@pkharat like the [previous comment](https://github.com/oneconvergence/group-based-policy/pull/358#discussion_r65281471), you are talking about
in lb_constants, add:
CREATE = 'create'
and change this line 587 to:
if operation == lb_constants.CREATE:

Am I correct?